reana_server/rest/workflows.py

```diff
@@ -341,6 +345,9 @@ def get_workflows(user, **kwargs):  # noqa
         search = request.args.get("search")
         sort = request.args.get("sort", "desc")
         status = request.args.getlist("status")
+        include_last_command = json.loads(
+            request.args.get("include_last_command", "false").lower()
+        )
```

Comment:
What about validating and parsing parameters with `use_args`, as done in the other endpoint?